src/pages/Defi/hooks/useEarnBalances.tsx

```diff
@@ -31,12 +27,11 @@ export function useEarnBalances(): UseEarnBalancesReturn {
   )
 
   const totalEarningBalance = bnOrZero(stakingOpportunitiesFiatBalance)
-    .plus(foxyBalancesData?.totalBalance ?? '0')
     .plus(foxEthLpFiatBalance ?? 0)
     .toString()
 
   return {
     totalEarningBalance,
-    loading: isFoxyBalancesLoading,
+    loading: false,
```

Comment:
This hook goes away entirely after https://github.com/shapeshift/web/pull/3802 so no point to remove this property now, this will do the trick